accelerator/xpu_accelerator.py

```diff
@@ -191,31 +192,31 @@ def supported_dtypes(self):
 
     @property
     def BFloat16Tensor(self):
-        return torch.xpu.BFloat16Tensor
+        return functools.partial(torch.tensor, dtype=torch.bfloat16, device='xpu')
```

Comment:
```suggestion
        return functools.partial(torch.tensor, dtype=torch.bfloat16, device=self.name)
```

Reply:
Thanks for the suggestion, already applied